doc/design.rst

```diff
@@ -0,0 +1,134 @@
+===============
+Software Design
+===============
+
+.. contents:: `Contents`
+    :depth: 3
+    :local:
+
+---------------------------
+Design for mlx.traceability
+---------------------------
+
+.. item:: DESIGN-TRACEABILITY Top level design for mlx.traceability
+    :fulfills: RQT-TRACEABILITY
+
+    Since the sphinx documentation system is python based, and it allows for plugin development in seperate
+    python packages, python was chosen. No limit should exist on the version of python or sphinx.
+
+.. item:: DESIGN-ITEMIZE Allow splitting the documentation in parts
+    :depends_on: DESIGN-TRACEABILITY
+    :fulfills: RQT-ITEMIZE
+
+    A directive name `item` is added to sphinx through the plugin that allows for splitting the documentation
```

Comment:
`Allows for splitting` -> `allows splitting`

Reply:
rewording in 792c66b